README.md

```diff
@@ -4,10 +4,30 @@ Welcome to the Chengdu chapter of [ProtoSchool](https://proto.school)!
 
 We are a group of people passionate about the decentralized web and excited to learn, teach, and share ideas. Everyone is welcome!
 
+We focus on building IPFS ecosystem (Decentralized Storage Network) cooperate with [Protocol Labs](https://protocol.ai/).  
+
+Everyone is welcome to join us to learn, to preach, to research, to develop, to share ideas of the amazing tech together!   
+:raised_hands:
+## Preliminaries
+
+Everyone should read Code of Conduct below firstly ！
+
+Everyone should follow this [Chapter Chengdu Slides Template](https://docs.google.com/presentation/d/1itLIYBr_z9MgsueJFT2ckAJs56Xuq3T-LlG59kQljb4/edit)
```

Comment:
This link appears to go to a document that is not publicly available. Could that be changed?